packages/framework/azure-client/src/AzureClient.ts

```diff
@@ -31,11 +32,28 @@ import { AzureUrlResolver } from "./AzureUrlResolver";
  */
 export class AzureClient {
     public readonly documentServiceFactory: IDocumentServiceFactory;
+    private containerList: Record<string, Container>;
 
     constructor(private readonly connectionConfig: AzureConnectionConfig) {
         this.documentServiceFactory = new RouterliciousDocumentServiceFactory(
             this.connectionConfig.tokenProvider,
         );
+        this.containerList = {};
+    }
+
+    public async attach(
+        id: string,
```

Comment:
this can chance to passing in the FluidContainer after #7162 is merged. Future API will be simply passing container in, but for now we need the ID to match it up.